src/payment/checkout/Checkout.jsx

```diff
@@ -231,34 +231,35 @@ class Checkout extends React.Component {
 
     return (
       <>
-        <div className={basketClassName}>
-          <h5 aria-level="2">
-            <FormattedMessage
-              id="payment.select.payment.method.heading"
-              defaultMessage="Select Payment Method"
-              description="The heading for the payment type selection section"
-            />
-          </h5>
-
-          <p className="d-flex flex-wrap">
-            <button type="button" className="payment-method-button active">
-              <img
-                src={AcceptedCardLogos}
-                alt={intl.formatMessage(messages['payment.page.method.type.credit'])}
+        { !this.props.isSubscription ? (
+          <div className={basketClassName}>
+            <h5 aria-level="2">
+              <FormattedMessage
+                id="payment.select.payment.method.heading"
+                defaultMessage="Select Payment Method"
+                description="The heading for the payment type selection section"
+              />
+            </h5>
+
+            <p className="d-flex flex-wrap">
+              <button type="button" className="payment-method-button active">
+                <img
+                  src={AcceptedCardLogos}
+                  alt={intl.formatMessage(messages['payment.page.method.type.credit'])}
+                />
+              </button>
+
+              <PayPalButton
+                onClick={this.handleSubmitPayPal}
+                className={classNames('payment-method-button', { 'skeleton-pulse': loading })}
+                disabled={submissionDisabled}
+                isProcessing={payPalIsSubmitting}
               />
-            </button>
-
-            <PayPalButton
-              onClick={this.handleSubmitPayPal}
-              className={classNames('payment-method-button', { 'skeleton-pulse': loading })}
-              disabled={submissionDisabled}
-              isProcessing={payPalIsSubmitting}
-            />
-
-            {/* Apple Pay temporarily disabled per REV-927  - https://github.com/openedx/frontend-app-payment/pull/256 */}
-          </p>
-        </div>
 
+              {/* Apple Pay temporarily disabled per REV-927  - https://github.com/openedx/frontend-app-payment/pull/256 */}
+            </p>
+          </div>
+        ) : null}
```

Comment:
Is there a benefit to using the ternary operator vs. `&&` here?  Is there a time when `isSubscription` could be undefined?

Reply:
No.
I know there will be no issue to use `&&` in our case but just following this practice when rendering [JSX conditionally](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx). Please let me know if you still want me to change it with `&&`

Reply:
Sounds good to keep as is.